controller/image.yaml

```diff
@@ -86,36 +94,36 @@ modules:
           - name: jboss.eap.cd.jolokia
           - name: os-eap7-openshift
           - name: jboss.eap.config.openshift
-            version: "16.0-openjdk11"
-          - name: jboss.eap.cd.openshift.modules
-          - name: os-eap7-ping
+            version: "EAP7.3.0-kie"
+          - name: jboss.container.eap.openshift.modules
```

Comment:
I think this one can be removed as well.
Do you mind to remove and see if things will work? On ocp you might need to test scaling a image up and see if the clustering view is correctly updated. (maybe KIE Server might be the best one for testing it).

If it works, please update all other image.yaml files by removing this module.